templates/machine_v1.go

```diff
@@ -45,15 +44,15 @@ func Create(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 
-	com := new(MachineTemplate)
-	err = json.Unmarshal(body, com)
+	var template *MachineTemplate
+	err = json.Unmarshal(body, &template)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 
-	db.Save(com.Name, com)
+	SaveTemplate(template)
 
-	w.Header().Set("Location", r.URL.Path+"/"+com.Name)
+	w.Header().Set("Location", r.URL.Path+"/"+template.Name)
```

Comment:
Can we attempt to create a unified path?

```
/tsg/template
/vpc/foo
```

That way we can switch on prefixes using fabio in the future.